### DISCUSSION TITLE
running skeema diff: what am I doing wrong

### DISCUSSION THREAD

**Discussion by Jengym1** [skeema/skeema#235] [Q&A] [answered]
I was able to run skeema init successfully in Ubuntu for Windows.   When I run skeema diff, I get some access errors, specifically access denied for user admin@xxx   The xxx is an IP.  I don't know where it is getting that.  It's not my IP, or the IP of the database.  I probably need to set up a user somehow, but I couldn't not find details on that.  Are you able to help?

![image](https://user-images.githubusercontent.com/126018846/220453634-15df8b19-b049-415b-bfbf-7809e16b3c4a.png)
[IssueConnectingToDbOnDiff_Skeema_Feb212023.docx](https://github.com/skeema/skeema/files/10797480/IssueConnectingToDbOnDiff_Skeema_Feb212023.docx)

**Comment by evanelias** [comment, marked as answer]
> (using password: NO)

The issue appears to be that you're not providing a password on your calls to `skeema diff`. I'm guessing you provided one for `skeema init` on the command-line manually? You either have to do that every time on all commands, or store a password in ~/.my.cnf, or use the $MYSQL_PWD env variable. It's just like the standard `mysql` client in that regard.

**Comment by Jengym1** [comment]
Thank you so much for getting back to me;  that was it.

**Comment by evanelias** [comment]
👍 awesome glad that was it! Will improve the docs on this soon, since in retrospect the [Getting Started tutorial](https://www.skeema.io/docs/examples/) only talks about passwords for `skeema init` but doesn't clarify that you still need to supply it for subsequent commands
